src/iort_lib/iort.cpp

```diff
@@ -91,6 +70,17 @@ inline bool inline_query(const std::string& query_string, Json::Value& ret,
     return reader.parse(r.text, ret);
 }
 
+void on_connect(struct mosquitto *mosq, void *obj, int rc) {
+    if (rc) {
+        exit(-1);
+    }
+    mosquitto_subscribe(mosq, NULL, ((Subscriber *)obj)->getTopic().c_str(), 0);
+}
+
+void on_message(struct mosquitto *mosq, void *obj, const struct mosquitto_message *msg) {
+    ((Subscriber *)obj)->messageCallback((char *)msg->payload);
```

Comment:
you should be able to use something like `std::to_string(msg->payload)` here to make it a C++ string